src/System.Data.Common/src/System/Data/Common/DbCommand.netcoreapp.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.Data.Common
+{
+    public abstract partial class DbCommand : IAsyncDisposable
+    {
+        public virtual Task PrepareAsync(CancellationToken cancellationToken = default)
+        {
+            if (cancellationToken.IsCancellationRequested)
+            {
+                return Task.FromCanceled(cancellationToken);
+            }
+            else
```

Comment:
Redundant else

Reply:
Will clean up tomorrow, thanks.

Reply:
Done.

Reply:
Will do the same for the rest of the project in a separate PR.